src/app/lib/utilities/passport/index.js

```diff
@@ -0,0 +1,38 @@
+import pathOr from 'ramda/src/pathOr';
+
+export const getPassportHome = pageData => {
+  const homeServiceUrl = pathOr(
+    null,
+    ['metadata', 'passport', 'home'],
+    pageData,
+  );
+  const passportHomeOverride = pathOr(
+    null,
+    ['metadata', 'passportHome'],
+    pageData,
+  );
+  return (
+    passportHomeOverride ||
+    (pageData && homeServiceUrl
+      ? homeServiceUrl
+          .split('/')
+          .slice(-1)
+          .pop()
+          .toLowerCase()
+      : null)
+  );
+};
+
+export const isValidPassportHome = (passportHome, service) => {
+  if (!passportHome) return true;
+
+  const overrides = {
+    portuguese: ['brasil'],
```

Comment:
Could/Should this live in the service config instead of building up a custom list here?

Reply:
This was my first thought @hindsc52, but @dr3's explained that we can't/shouldn't change the config files in the data folder, because they have to match with what we receive from ares.

Reply:
This sounds like confusion between the service configs ( /src/app/lib/config/services ) and the fixture data ( /data )

Reply:
Agree with @jimjohnsonrollings `/src/app/lib/config/services` is for us to configure per service.

Reply:
Are we okay with something like:
```
const service = {
  default: {
    ...
    passportHomes: [ 'brasil' ],
    ...
  }
}
```

in `portuguese.js` ?

Reply:
Does this need to be an array @mykeels ?

Reply:
From [this comment](https://github.com/bbc/simorgh/pull/3773#discussion_r324658395), it looks like there could be multiple passport homes that map to the portuguese service.

Reply:
I would base off of this comment and we can change in the future if we need to https://github.com/bbc/simorgh/pull/3773#discussion_r324690582